### ISSUE TITLE
9 Quotes by Tony Robbins That Will Help Rewire Your Brain

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#73]
# TL;DR
You can do anything when you rewire your brain, believe anything is possible, and you control the meaning of what happens in your life.

# Key Takeaways

- **“Nothing in life has any meaning except the meaning we give it.”** Your brain gives meaning to every thought you have. That’s right: you decide the meaning of everything in life. Someone can tell you their opinion but you decide if you are going to accept it or not.
- **“We can change our lives. We can do, have, and be exactly what we wish.”** The truth is you can be anything you want. You can dream something up out of nowhere and then use your daily actions to manifest it into the physical world. The key concept here is that we are responsible.
- **“Every man dies. Not every man lives.”**. You’ve met that man or woman who is dead inside. They show up every day at a job they hate, to pay for a house they don’t need, to buy stuff they think will make them happy, and the entire time they treat most people badly.
- **“A real decision is measured by the fact that you’ve taken a new action. If there’s no action, you haven’t truly decided.”**. There are times where we sit around and realize that we haven’t taken the actions we committed to. Understanding why that is can be challenging.
- **“If you talk about it, it’s a dream, if you envision it, it’s possible, but if you schedule it, it’s real.”**. The moment you think of an action you need to take, either take action right then, or if that’s not possible, schedule that action in your calendar.
- **“The only thing that’s keeping you from getting what you want is the story you keep telling yourself.”**. They can tell their story about what they never got started with, with pinpoint precision. They can blame the results they’ve never got on a divorce, their kids, a job they hate, or toxic friends. The story you tell yourself about why you can’t have that thing is the barrier.
- **“Focus on where you want to go, not on what you fear.”**. Think about where you’re trying to head. What would life look like if it made sense? What direction have you always wanted to go and never gone?
- **“I’ve found that 20 percent of any change is knowing how, but 80 percent is knowing why. ”**. The “why” is the hardest bit. It takes a lot of time to understand your motivations and decide on why you do what you do.
- **“At any moment, the decision you make can change the course of your life forever.”**. You never know which decision you will make that will change your life forever.

# Links
- https://medium.com/mind-cafe/9-quotes-by-tony-robbins-that-will-help-rewire-your-brain-9220e03bca87
